ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/ScanConfigDetailPage.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useCallback } from 'react';
+import { generatePath, Link, useParams } from 'react-router-dom';
+import { Button, Divider, PageSection } from '@patternfly/react-core';
+
+import { complianceEnhancedScanConfigDetailPath } from 'routePaths';
+import useRestQuery from 'hooks/useRestQuery';
+import { getScanConfig } from 'services/ComplianceEnhancedService';
+import ViewScanConfigDetail from './ViewScanConfigDetail';
+
+const EditScanConfigButton = (id) => {
```

Comment:
```suggestion
const EditScanConfigButton = (id: string) => {
```
Nit to remove implicit `any`

Reply:
Agree, but this is actually an artifact that isn't used in this file anymore; I moved it to `ViewScanConfigDetail.tsx`. I'll delete it here.